File: Problem Solutions/Interviewbit/Math/FizzBuzz.cpp
vector<string> Solution::fizzBuzz(int A) {
  vector<string> res;
  for (int i = 1; i <= A; ++i) {
    if (i % 3 == 0 && i % 5 == 0) {
      res.push_back("FizzBuzz");
    } else if (i % 3 == 0) {
      res.push_back("Fizz");
    } else if (i % 5 == 0) {
      res.push_back("Buzz");
    } else {
      res.push_back(to_string(i));
    }
  }
  return res;
}

File: Problem Solutions/LeetCode/Misc/fizzbuzz.py
"""
Write a program that outputs the string representation of numbers from 1 to n.
But for multiples of three it should output “Fizz” instead of the number and 
for the multiples of five output “Buzz”. For numbers which are multiples of both three and five output “FizzBuzz”.

Example:
n = 15,
Return:
[
    "1",
    "2",
    "Fizz",
    "4",
    "Buzz",
    "Fizz",
    "7",
    "8",
    "Fizz",
    "Buzz",
    "11",
    "Fizz",
    "13",
    "14",
    "FizzBuzz"
]
"""

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []
        for i in range(1, n+1):
            if i % 5 == 0 and i % 3 == 0:
                res.append("FizzBuzz")
            elif i % 3 == 0:
                res.append("Fizz")
            elif i % 5 == 0:
                res.append("Buzz")
            else:
                res.append(str(i))
        return res
        